docs/examples/prometheus_exemplars/README.rst

```diff
@@ -0,0 +1,107 @@
+======================================================
+Prometheus Metric Exemplars with OpenTelemetry Tracing
+======================================================
+
+The full code for this example is available `on Github
+<https://github.com/GoogleCloudPlatform/opentelemetry-operations-python/tree/main/docs/examples/prometheus_exemplars>`_.
+
+This end-to-end example shows how to instrument a Flask app with with `Prometheus
+<https://prometheus.io/>`_ metrics linked to OpenTelemetry traces using exemplars. The example
+manually adds exemplars to a Prometheus Histogram which link the metric in Cloud Monitoring to
+Spans in Cloud Trace.
+
+OpenTelemetry Python is configured to send traces to the `OpenTelemetry Collector
+<https://opentelemetry.io/docs/collector/>`_ and the Collector scrapes the python server's
+Prometheus endpoint. The Collector is configured to send metrics to `Google Cloud Managed
+Service for Prometheus <https://cloud.google.com/stackdriver/docs/managed-prometheus>`_ and
+traces to `Google Cloud Trace <https://cloud.google.com/trace/docs/overview>`_.
+
+To run this example you first need to:
+    * Create a Google Cloud project. You can `create one here <https://console.cloud.google.com/projectcreate>`_.
+    * Enable Cloud Trace API (listed in the Cloud Console as Stackdriver Trace API) in the project `here <https://console.cloud.google.com/apis/library?q=cloud%20trace&filter=visibility:public>`_. If the page says "API Enabled" then you're done! No need to do anything.
+    * Enable Default Application Credentials by creating setting `GOOGLE_APPLICATION_CREDENTIALS <https://cloud.google.com/docs/authentication/getting-started>`_ or by `installing gcloud sdk <https://cloud.google.com/sdk/install>`_ and calling ``gcloud auth application-default login``.
```

Comment:
It's "Application Default Credentials." I'd suggest giving them one way to do it which covers the 80% case (probably gcloud auth application-default login) and pointing them to the GCP docs for alternatives.

Reply:
No problem